spec/amber/environment/env_spec.cr

```diff
@@ -24,7 +24,7 @@ module Amber::Environment
         amber_env.{{env.id}}.should_not be_nil
       end
     end
-  {% end %}
+    {% end %}
```

Comment:
Is this meant to be moved over?

Reply:
Yes, to align the with the opening if. 

I just noticed that this file is named incorrectly